packages/server/src/modules/subscription/resolvers.js

```diff
@@ -119,7 +118,7 @@ export default pubsub => ({
           await stripe.customers.deleteSource(stripeCustomerId, stripeSourceId);
         } catch (e) {
           const e = new FieldError();
-          e.setError('subscription', 'Error cancelling subscription.');
+          e.setError('subscription', context.req.headers.cookie, 'user', 'cancelSubscription');
```

Comment:
We should avoid adding sophisticated usage patterns like this into the kit. I would rather want usage like:
`const e = new FieldError(context.req, 'user');`
`e.setError('subscription', 'cancelSubscription');`

Please think very carefully what usage patterns you introduce to the code base. It is very important

Reply:
Fixed.